templates/stitch_images.ipynb

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
<p>Get max size automatically, cols should be sqrt(len(fovs))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>oops yeah forgot to update these inputs </p>